middleware/auth.js

```diff
@@ -0,0 +1,27 @@
+require("dotenv");
+const jwt = require("jsonwebtoken");
+const User = require("../models/user.model");
+const secret = process.env.SECRET;
+
+const createToken = async (data) => {
+  return jwt.sign({ id: data }, secret, { expiresIn: "10h" });
+};
+
+const validateToken = async (req, res, next) => {
+  try {
+    const cookie = req.cookies.userId;
+    if (!cookie) return res.sendStatus(401);
+    req.user = await findUserByToken(cookie);
```

Comment:
not good enough, now I can add any cookie that I want from the browser and it will not fail. 
I want you to check if user exists in the system, if not - throw an error